ui/widgets/nuxeo-selectivity.js

```diff
@@ -3464,8 +3464,12 @@ typedArrayTags[weakMapTag] = false;
         _updateInputWidth() {
           const inputContent =
             this.input.value || (!this._data.length && this.options.placeholder) || '';
-          this.input.setAttribute('size', inputContent.length + 2);
-
+            const placeholderLength= typeof this.options.placeholder!== 'undefined' && this.options.placeholder.length;
```

Comment:
Took reference from here for checking `undefined ` string.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined

Reply:
To keep the code consistent, please use spacing between the operators and the vars
```suggestion
            const placeholderLength = typeof this.options.placeholder !== 'undefined' && this.options.placeholder.length;
```